Library/Homebrew/cleanup.rb

```diff
@@ -350,10 +359,12 @@ def cache_files
       files = cache.directory? ? cache.children : []
       cask_files = (cache/"Cask").directory? ? (cache/"Cask").children : []
       api_source_files = (cache/"api-source").glob("*/*/*/*/*") # org/repo/git_head/type/file.rb
```

Comment:
Any reason why this isn't just
```suggestion
      api_source_files = (cache/"api-source").glob("**/*.rb") # org/repo/git_head/type/file.rb
```
?

Reply:
@carlocab Try it: if it works, it seems better.

Reply:
Parts of the cleanup code assume 5 levels and could break if there's a file out of place, so it is safer to have it as `*/*/*/*/*`.